### PR TITLE
Add Croatian translation

### PR DESCRIPTION
**Pull request by deadly667**
Hi,

I added Croatian translation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ This card supports translations. Please, help to add more translations and impro
 
 <!-- Languages are sorted alphabetically -->
 - English
+- Hrvatski (Croatian)
```
Comment (MrBartusek) [inline README.md:66]: Languages should be sorted alphabetically

File: src/translations/hr.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  },
+  "editor": {
+    "entity": null,
+    "integration": null,
+    "required": null,
+    "automatic": null,
+    "recommended": null,
+    "override_headline": null,
+    "hide_when_no_warning": null
+  },
```
Comment (MrBartusek) [inline src/translations/hr.json:25]: Can you also translate these missing keys marked by `null`?

**Review by MrBartusek — CHANGES_REQUESTED** [review]

**Comment by deadly667** [comment]
Done. I saw that but then I saw that also English is first and thats not sorted alphabetically anyhow so I was confused. Still I think that then English should be corrected also.

**Comment by MrBartusek** [comment]
> Done. I saw that but then I saw that also English is first and thats not sorted alphabetically anyhow so I was confused. Still I think that then English should be corrected also.

English is just not sorted cause its the primary language

**Comment by deadly667** [comment]
Maybe to put some message about that. That English is first intentionally and that all other languages should be in alphabetical order. Because its misleading in this way.

**Review by MrBartusek — APPROVED** [review]

**Comment by MrBartusek** [comment]
Thanks!

**Comment by MrBartusek** [comment]
> Maybe to put some message about that. That English is first intentionally and that all other languages should be in alphabetical order. Because its misleading in this way.

That's actually a valid point, i have changed that https://github.com/MrBartusek/MeteoalarmCard/commit/71fe6df3a4d3a39ac2378c3b18585d4a7eacdffd

**Pull request merged**